spec/factories/provider.rb

```diff
@@ -15,4 +15,5 @@
   end
 
   factory :provider_openstack, :class => "ManageIQ::Providers::Openstack::Provider", :parent => :provider
+  factory(:provider_ansible_tower, :class => "ManageIQ::Providers::AnsibleTower::Provider", :parent => :provider) { url "example.com" }
```

Comment:
Can you just multiline this?  The line is already long enough, so mtuliline would make it clearer.